src/high-level/util.lisp

```diff
@@ -0,0 +1,97 @@
+;;;; util.lisp
+;;;;
+;;;; Author: Cole Scott
+
+(in-package #:magicl)
+
+(defconstant +float-comparison-threshold+  single-float-epsilon)
```

Comment:
Final request about this. Could you please redefine these thresholds as special variables (with `defvar`) rather than constants and change their names so that they're sandwiched between `*` instead of `+`? The reason is that it comes in handy to be able to override their values within a `let` form.

Reply:
FIxed in 9c4e1ac